modules/swagger-codegen/src/main/resources/csharp/Project.mustache

```diff
@@ -88,7 +88,10 @@
     {{/netStandard}}
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs"/>
+    <Compile Include="Api\**\*.cs"/>
```

Comment:
My test results also look good (I'd to reinstall mono in my Mac as the commands couldn't be found)